DrinkMacroCreator.lua

```diff
@@ -14,184 +14,241 @@ Drinks Table (haha) Notes:
+    30457, -- Gilneas Sparkling Water ...................... (2k MP) [94]
+    40357, -- Grizzleberry Juice ........................... (2k MP) [94]
+    44750, -- Mountain Water ............................... (2k MP) [94]
+    34780, -- Naaru Ration ................................. (2k MP) [94]
+    35954, -- Sweetened Goat's Milk ........................ (2k MP) [94]
+    37253, -- Frostberry Juice ............................. (2k MP) [94]
+    32453, -- Star's Tears ................................. (2k MP) [94]
+    32668, -- Dos Ogris .................................... (2k MP) [94]
+    33042, -- Black Coffee ................................. (2k MP) [94]
+    38431, -- Blackrock Fortified Water .................... (2k MP) [94]
+    61382, -- Garr's Limeade ............................... (2k MP) [80]
+    32455, -- Star's Lament ................................ (2k MP) [72]
+    18300, -- Hyjal Nectar ................................. (2k MP) [72]
+    33053, -- Hot Buttered Trout ........................... (1574 MP, 1638HP) [52]
+    38429, -- Blackrock Spring Water ....................... (1k MP) [45]
+    8766, -- Morning Glory Dew ............................. (1k MP) [45]
+    63530, -- Refreshing Pineapple Punch ................... (504 MP) [24]
+    90659, -- Jasmine Tea .................................. (504 MP) [24]
+    1179, -- Ice Cold Milk ................................. (504 MP) [24]
+    17404, -- Blended Bean Brew ............................ (504 MP) [24]
+    49601, -- Volcanic Spring Water ........................ (504 MP) [24]
+    49602, -- Earl Black Tea ............................... (504 MP) [24]
+    1645, -- Moonberry Juice ............................... (588 MP) [19]
+    63023, -- Sweet Tea .................................... (588 MP) [19]
+    19300, -- Bottled Winterspring Water ................... (588 MP) [19]
+    1708, -- Sweet Nectar .................................. (324 MP) [12]
+    4791, -- Enchanted Water ............................... (324 MP) [12]
+    10841, -- Goldthorn Tea ................................ (324 MP) [12]
+    1205, -- Melon Juice ................................... (288 MP) [12]
+    9451, -- Bubbling Water ................................ (288 MP) [12]
+    19299, -- Fizzy Faire Drink ............................ (288 MP) [12]
+    90660, -- Black Tea .................................... (288 MP) [12]
+    155909, -- Bottled Stillwater .......................... (288 MP) [12]
+    1401, -- Riverpaw Tea Leaf ............................. (115 MP, 60 HP) [12]
+    159, -- Refreshing Spring Water ........................ (180 MP) [10]
+    49254, -- Tarp Collected Dew ........................... (180 MP) [10]
+    60269 -- Well Water ................................... (180 MP) [10]
 }
 
+-- Merge the two tables
+table.move(drinks, 1, #drinks, #magefood + 1, magefood)
+for i = 1, #magefood do
+    drinks[i] = magefood[i]
+end
+
```

Comment:
WoW's lua doesn't have table.move as far as i can see? Could you change this to something else please.

```suggestion
for i = #magefood, 1, -1 do
    table.insert(drinks, 1, magefood[i])
end
```